src/components/VExpansionPanel/VExpansionPanelContent.spec.js

```diff
@@ -20,4 +20,14 @@ test('VExpansionPanelContent.js', ({ mount }) => {
 
   //   expect(wrapper.html()).toMatchSnapshot()
   })
+
+  it('should render an expanded component with lazy prop and match snapshot', () => {
```

Comment:
is this meant to be commented out?

Reply:
Yes, similar to other tests for this components. It uses "inject" which as I understand is not supported yet in avoriaz